.github/workflows/check-bump-interface-version.yml

```diff
@@ -1,12 +1,13 @@
-name: 'check for changes that should accompany env.rs change'
+name: 'check for changes that should accompany env.json change'
 
 on:
   pull_request:
     paths:
-      - 'soroban-env-common/src/env.rs'
+      - 'soroban-env-common/env.json'
 
 jobs:
   check:
+    if: '!contains( github.event.pull_request.labels.*.name, "skip-env-interface-version-bump")'
```

Comment:
@stellar/contract-committers I'm adding a label we can use to skip bumping the env interface version in situations where we know for certain it does not need bumping.

I'm adding this because I'm seeing moments where we are needing to bump it because we're touching a file, but not actually changing anything meaningful in it.

In the case of this PR I plan to use the label because I moved the interface from env.rs into env.json, but the contents are identical.